src/Umbraco.Web/Editors/EntityController.cs

```diff
@@ -250,6 +308,64 @@ public IEnumerable<EntityBasic> GetByIds([FromUri]int[] ids, UmbracoEntityTypes
             return GetResultForIds(ids, type);
         }
 
+        /// <summary>
+        /// Get entities by GUID ids
+        /// </summary>
+        /// <param name="ids"></param>
+        /// <param name="type"></param>
+        /// <returns></returns>
+        /// <remarks>
+        /// We allow for POST because there could be quite a lot of Ids
+        /// </remarks>
+        [HttpGet]
+        [HttpPost]
+        public IEnumerable<EntityBasic> GetByIds([FromJsonPath]Guid[] ids, UmbracoEntityTypes type)
+        {
+            if (ids == null)
+            {
+                throw new HttpResponseException(HttpStatusCode.NotFound);
+            }
+            return GetResultForKeys(ids, type);
+        }
+
+        /// <summary>
+        /// Get entities by string ids - will try to convert to the correct id type (int, guid, udi)
+        /// </summary>
+        /// <param name="ids"></param>
+        /// <param name="type"></param>
+        /// <returns></returns>
+        /// <remarks>
+        /// We allow for POST because there could be quite a lot of Ids
+        /// </remarks>
+        [HttpGet]
+        [HttpPost]
+        public IEnumerable<EntityBasic> GetByIds([FromJsonPath]Udi[] ids, [FromUri]UmbracoEntityTypes type)
+        {
+            if (ids == null)
+            {
+                throw new HttpResponseException(HttpStatusCode.NotFound);
+            }
+
+            if (ids.Length == 0)
+            {
+                return Enumerable.Empty<EntityBasic>();
+            }
+
+            //all udi types will need to be the same in this list so we'll determine by the first
+            //currently we only support GuidIdi for this method
+
+            var guidUdi = ids[0] as GuidUdi;

```

Comment:
Again I know its our own AngularJS Services that should be posting to this API endpoint, but as this method requires to use only UDI.Guids and of the same UDI entity type I think the XML doc for this method needs to mention this for auto API generated docs, as you will only discover this if you were to read this comment in the source code.